### PR TITLE
Old quickbms patchers deleted, only new mangetsu based patchers kept.

### PR DESCRIPTION
**Pull request by harphield**
Old quickbms patchers deleted, only new mangetsu based patchers kept.
Patcher Readme updated.

### CODE REVIEW

File: tools/patcher/README.md [hunk 1]
```diff
@@ -1,33 +1,47 @@
...
+to the readme there.
 
-Then, for Linux, download compressonatorcli into the **compressonator** folder: https://github.com/GPUOpen-Tools/compressonator/releases/tag/V4.2.5185 -
-this tools builds .dds files from .png files to be inserted. **ALSO**, edit the compressonatorcli script and remove the first empty line before ``#!/bin/bash``.
+Download compressonatorcli: https://github.com/GPUOpen-Tools/compressonator/releases/tag/V4.2.5185 - and rename the ``compressonatorcli`` script to ``compressonator``.
+This tools builds .dds files from .png files to be inserted. **ALSO**, edit the script and remove the first empty line before ``#!/bin/bash``.
 
-For Windows, just install the CompressonatorCLI software, this script uses the default install path.
+The binaries need to be in your PATH system variable for it to work. To do that, edit your ``~/.bashrc``
+file and add them at the end of the file, like so:
```
Comment (rschlaikjer) [inline tools/patcher/README.md:23]: Maybe just add the entire build folder to $PATH instead of having to do it one by one?

Comment (harphield) [reply tools/patcher/README.md:23]: Hmm you're right, reading a bit more about all that PATH adding now and that should work better.

Comment (harphield) [reply tools/patcher/README.md:23]: @rschlaikjer should be better now.

File: tools/patcher/README.md [hunk 2]
```diff
@@ -1,33 +1,47 @@
...
 
 ## 4. Copy to mod folder
 If all went well, you should have your new allui.* files in **_new_mrgs**. Have fun!
 
 ## 5. Do it again
 If you need to rebuild your allui.mrg again, because new stuff has been translated / changed,
-you need to delete the .dds files that you want updated from the **_replace** folder (or just delete the
-whole folder, but this will make the script to redo all the conversions again).
\ No newline at end of file
+you need to delete the .dds files that you want updated from the **.user_interface_dds** folder (or just delete the
```
Comment (rschlaikjer) [inline tools/patcher/README.md:42]: This shouldn't be necessary - the script should `stat` the outputs and rebuild if they are older than the source images

**Pull request merged**